azurerm/modules/azurerm-adb/var.tf

```diff
@@ -109,4 +126,130 @@ variable "enable_sql_access" {
   type        = bool
   description = "Whether to enable sql access for the databricks group"
   default     = true
+}
+
+variable "nat_idle_timeout" {
+  type        = number
+  default     = 10
+  description = "Idle timeout period in minutes."
+}
+
+############################################
+# Network Details
+############################################
+
+variable "create_subnets" {
+  type        = bool
+  default     = false
+  description = "Set to true if you need the module to create the subnets for you."
+}
+
+variable "create_pe_subnet" {
+  type        = bool
+  default     = false
+  description = "Set to true if you need the module to create the private endpoint subnet."
+}
+
+variable "create_db_dns_zone" {
+  type        = bool
+  default     = true
+  description = "Create DNS Zone for Azure Databricks."
+}
+
+variable "db_dns_zone_rg" {
+  type        = string
+  default     = "value"
+  description = "Resource Group where DNS is created."
+}
+
+variable "vnet_name" {
+  type        = string
+  default     = ""
+  description = "Name of the VNET inwhich the Databricks Workspace will be provisioned."
+}
+
+variable "vnet_resource_group" {
+  type        = string
+  default     = ""
+  description = "The Resource Group which the VNET is provisioned."
+}
+
+variable "public_subnet_name" {
+  type        = string
+  default     = ""
+  description = "Name of the Public Databricks Subnet."
+}
+
+variable "private_subnet_name" {
+  type        = string
+  default     = ""
+  description = "Name of the Private Databricks Subnet."
+}
+
+variable "public_subnet_prefix" {
+  type        = list(string)
+  default     = []
+  description = "IP Address Space fo the Public Databricks Subnet."
+}
+
+variable "private_subnet_prefix" {
+  type        = list(string)
+  default     = []
+  description = "IP Address Space fo the Private Databricks Subnet."
+
+}
+
+variable "pe_subnet_prefix" {
+  type        = list(string)
+  default     = []
+  description = "IP Address Space fo the Private Endpoints Databricks Subnet."
```

Comment:
```suggestion
  description = "IP Address Space for the Private Endpoints Databricks Subnet."
```

Reply:
Closing this PR. Its been handled by another PR now.